_data/versions.yml

```diff
@@ -1,17 +1,43 @@
 v3.0:
-- title: v3.0.0
+- title: v3.0.1
   note: |
-    21 December 2017
+    22 December 2017
 
     #### What's new
+
+    > **Important** This release includes breaking changes introduced as part of Calico v3.0.
+    > If you are upgrading a previous Calico deployment to Calico v3.0 you
+    > must follow this one-time migration process as part of the
+    > [upgrade process](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/).
+    > Please carefully review the upgrade preocess before attempting to upgrade
+    > to Calico v3.0.
+    {: .alert .alert-danger}
+
+    ##### Migration and upgrade from v2.6.5
+    
+    - This version of Calico supports [migration and upgrade](/v3.0/getting-started/kubernetes/upgrade/)
+      from Calico v2.6.5. 
+
+    - The Calico resource model has been updated which requires that Calico
+      manifests that are loaded using the `calicoctl` command must be updated or
+      converted to the new model.
```

Comment:
can we maybe say "...converted to the new model using `calicoctl convert` command"?